src/components/types.ts

```diff
@@ -1,4 +1,4 @@
-const SizeOptions = 'xs s m l xl 2xl'.split(' ');
+const SizeOptions = ['xs', 's', 'm', 'l', 'xl', '2xl'] as const;
```

Comment:
Why is this better?

Reply:
because: 
```
const SizeOptions = 'xs s m l xl 2xl'.split(' ');
export type Size = (typeof SizeOptions)[number]; // Size = string
``` 
while:
```
const SizeOptions = ['xs', 's', 'm', 'l', 'xl', '2xl'] as const;
export type Size = (typeof SizeOptions)[number]; // Size = 'xs' | 's' | 'm' | 'l' | 'xl' | '2xl'
```